siff-dr/siff-header.go

```diff
@@ -10,8 +10,8 @@ import (
 
 // SIFF constants
 const (
-	EXP  layers.IPv4Flag = 1 << 3
-	EVIL layers.IPv4Flag = 1 << 2 // http://tools.ietf.org/html/rfc3514 ;)
+	EXP  layers.IPv4Flag = 1 << 2
+	EVIL layers.IPv4Flag = 1 << 3 // http://tools.ietf.org/html/rfc3514 ;)
```

Comment:
The Go style seems to suggest that constants are [all caps](https://golang.org/doc/effective_go.html#constants).

But they're not actually being used in the flags field, I just maintained the conventions to not break the existing usage of the functions. It's actually being used as the position in one of the bytes in the first IPv4 option, used for the flags.